### ISSUE TITLE
[Bug] ObjectDisposedException when navigating to a not existing url

### ISSUE THREAD

**Issue by manureini** [Blazored/Modal#524]
**Describe the bug**

I'm getting an exception
```
Error: System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'Microsoft.JSInterop.Implementation.JSObjectReference'.
   at Microsoft.JSInterop.Implementation.JSObjectReference.InvokeAsync[TValue](String identifier, Object[] args)
   at Microsoft.JSInterop.JSObjectReferenceExtensions.InvokeVoidAsync(IJSObjectReference jsObjectReference, String identifier, Object[] args)
   at Blazored.Modal.BlazoredModal.ClearBodyStyles()
   at Blazored.Modal.BlazoredModal.CancelModals(Object sender, LocationChangedEventArgs e)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at Microsoft.AspNetCore.Components.Rendering.RendererSynchronizationContext.ExecuteSynchronously(TaskCompletionSource completion, SendOrPostCallback d, Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at Microsoft.AspNetCore.Components.Rendering.RendererSynchronizationContext.ExecuteBackground(WorkItem item)
```

**To Reproduce**
1. Use Blazor Server
2. Put this code on a .razor page:
```razor
<button type="button" @onclick="@(() => Navigation.NavigateTo("/notexistingpage"))">Test</button>
```
3. Click on the button

**Expected behavior**
No exception / ignore it.

**Hosting Model (is this issue happening with a certain hosting model?):**
 - Blazor Server

**Comment by chrissainty** [comment]
Is this still an issue @manureini? I can't seem to repro it.

**Comment by chrissainty** [comment]
I believe this was fixed by #417 so I'm going to close this as complete. Please let me know if you're still seeing issues.
